monkey/monkey_island/cc/ui/src/components/pages/MapPage.js

```diff
@@ -88,10 +88,11 @@ class MapPageComponent extends AuthComponent {
       {
         method: 'POST',
         headers: {'Content-Type': 'application/json'},
-        body: JSON.stringify({kill_time: Date.now()})
+        // Python uses seconds, Date.now uses milliseconds, so convert
+        body: JSON.stringify({kill_time: Date.now() / 1000})
```

Comment:
This may be more explicit:
```suggestion
        // Python uses floating point seconds, Date.now uses milliseconds, so convert
        body: JSON.stringify({kill_time: Date.now() / 1000.0})
```